source/standards/git.html.md.erb

```diff
@@ -72,6 +81,8 @@ You should leave a blank line before the rest of the commit message, which you
 should wrap at around 72 characters: this makes it easier to view commit
 messages in a terminal.
 
+Ignoring the established community guidelines for structuring commit messages
+is mildy anti-social to other git users.
```

Comment:
While I completely agree with this, I don't think it's necessary here. We have the motivation at the top of the document, the rest of the document should just be matter-of-fact statements about what is expected.

Reply:
Agreed 👍